snuba/datasets/plans/single_storage.py

```diff
@@ -56,7 +52,7 @@ def process_and_run_query(
         return process_and_run_query(query, request_settings)
 
 
-class SingleStorageQueryPlanBuilder(ClickhouseQueryPlanBuilder):
+class SingleStorageQueryPlanBuilder(QueryPlanBuilder):
```

Comment:
Is there a reason you didn't move these (this and SelectedStorageQueryPlanBuilder below) into the `clickhouse` module? Is there some other plan for them?